tests/integration/src/install_tests.rs

```diff
@@ -0,0 +1,961 @@
+use std::mem;
+
+use candid::{Encode, Principal};
+use pocket_ic::PocketIc;
+use rstest::rstest;
+use station_api::{
+    AccountResourceActionDTO, AllowDTO, AuthScopeDTO, InitAccountInput, InitAssetInput,
+    InitNamedRuleInput, InitPermissionInput, InitRequestPolicyInput, InitUserGroupInput,
+    MetadataDTO, PermissionResourceActionDTO, RequestPolicyRuleDTO, RequestSpecifierDTO,
+    ResourceActionDTO, ResourceDTO, ResourceIdDTO, SystemInit, SystemInstall, UserIdentityInput,
+    UserInitInput, UserResourceActionDTO, UserStatusDTO, UuidDTO,
+};
+use uuid::Uuid;
+
+use crate::{
+    setup::{get_canister_wasm, setup_new_env, WALLET_ADMIN_USER},
+    station_test_data::{
+        account::list_accounts, asset::list_assets, named_rule::list_named_rules,
+        permission::list_permissions, request_policy::list_request_policies, user::list_users,
+        user_group::list_user_groups,
+    },
+    utils::{await_station_healthy, ADMIN_GROUP_ID, OPERATOR_GROUP_ID},
+    TestEnv,
+};
+
+#[test]
+fn install_with_default_policies() {
+    let TestEnv {
+        env, controller, ..
+    } = setup_new_env();
+
+    let canister_id = env.create_canister_with_settings(Some(controller), None);
+
+    env.set_controllers(canister_id, Some(controller), vec![canister_id, controller])
+        .expect("failed to set canister controller");
+
+    env.add_cycles(canister_id, 5_000_000_000_000);
+    let station_wasm = get_canister_wasm("station").to_vec();
+    let upgrader_wasm = get_canister_wasm("upgrader").to_vec();
+
+    let asset_1_id = Uuid::new_v4().hyphenated().to_string();
+    let asset_2_id = Uuid::new_v4().hyphenated().to_string();
+
+    let account_1_id = Uuid::new_v4().hyphenated().to_string();
+
+    let users = vec![
+        UserInitInput {
+            identities: vec![UserIdentityInput {
+                identity: WALLET_ADMIN_USER,
+            }],
+            name: "station-admin".to_string(),
+            groups: None,
+            id: None,
+            status: None,
+        },
+        UserInitInput {
+            identities: vec![UserIdentityInput {
+                identity: Principal::from_slice(&[2; 29]),
+            }],
+            name: "inactive-station-admin".to_string(),
+            groups: Some(vec![ADMIN_GROUP_ID.hyphenated().to_string()]),
+            id: None,
+            status: Some(station_api::UserStatusDTO::Inactive),
+        },
+        UserInitInput {
+            identities: vec![UserIdentityInput {
+                identity: Principal::from_slice(&[3; 29]),
+            }],
+            name: "other-user".to_string(),
+            groups: Some(vec![]),
```

Comment:
What if an invalid group ID was specified here?

Reply:
It will fail with malformed UUID, or if it's valid UUID, it'll fail if it doesn't exist as a group

Reply:
Is there a test for this behavior?

Reply:
yes `test_initial_users_with_bad_groups`